pkg/alerts/alertsHandler/notificationHandler.go

```diff
@@ -143,6 +151,26 @@ func sendWebhooks(webhookUrl, subject, message string) error {
 	return err
 }
 
+func isSilencePeriodOver(alertID string) (bool, error) {
+	silencePeriod, lastSendTime, err := processGetSilencePeriodRequest(alertID)
+
+	if lastSendTime.IsZero() {
+		return true, nil
+	}
+
+	if err != nil {
+		return false, err
+	}
```

Comment:
@AndrewHess I agree, I think it's better to fail-safe in the context of an alert system. I've updated the function to return true in case of an error during the silence period.